packages/axios-extension/src/abap/app-index-service.ts

```diff
@@ -34,4 +35,34 @@ export abstract class AppIndexService extends Axios implements Service {
         const response = await this.get('/', { params });
         return JSON.parse(response.data).results as AppIndex;
     }
+
+    /**
+     * Gets the URL to the app manifest with the specified id.
+     *
+     * @param {string} appId - The id of the app.
+     * @returns {Promise<string>} - URL to the app manifest.
+     */
+    public async getManifestUrl(appId: string): Promise<string> {
+        let result = '';
+        const params = {
+            'id': appId
+        };
+        try {
+            const response = await this.get('/ui5_app_info_json', { params });
+            const responseValues: any = Object.values(JSON.parse(response.data));
```

Comment:
No any please.
Why do we check for `manifestUrl` or `manifest`? Is a different property used in different versions?

Reply:
We check for both due to historic reasons, as the name of the property varies in the manifest.json of some applications.
Fixed with: 0b3bbd403dd4a46cc48f14ac48822e9a55d4b136